workspace/assets/css/lib/class-will-change.less

```diff
@@ -0,0 +1,3 @@
+.will-change-transform {
+	will-change: transform;
```

Comment:
Can we also all the `will-change: -web-kit-transform;` version, before the standard one ?